Source/Eto/Forms/Taskbar.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Eto.Forms
+{
+	/// <summary>
+	/// Methods to interact with application taskbar button.
+	/// </summary>
+	[Handler(typeof(IHandler))]
+	public static class Taskbar
+	{
+		/// <summary>
+		/// Sets the state and progress of the application.
+		/// </summary>
+		/// <param name="state">Taskbar button state.</param>
+		/// <param name="progress">Progress in range from 0.0f to 1.0f.</param>
+		public static void SetProgress(TaskbarProgressState state, float progress)
```

Comment:
Should progress have a default of 0 here?  I assume it only makes sense to set its value when using TaskbarProgressState.ShowProgressbar